docs/ops/broadcast_rules.md

```diff
@@ -0,0 +1,85 @@
+# Broadcast Rules For Elementwise Operations {#openvino_docs_ops_broadcast_rules}
+
+The purpose of this document is to provide a set of common rules which are applicable for ops using broadcasting.
+
+## Description
+
+Broadcast allows to perform element-wise operation for inputs with different shapes. Smaller input is *stretched* into an array with the same shape as the larger input. As a result both inputs have compatible shapes.
+
+## Rules
+
+**None broadcast**:
+1. Input tensors dimensions must match.
+2. No implicit broadcast rule is applied.
+
+**Numpy broadcast**:
+1. Right aligned dimensions of the two tensors are compared elementwise.
+2. Smaller tensor is prepended with dimension(s) of size 1 in order to have the same shape as the larger tensor.
+3. After alignment two tensors are compatible when:
+   * both are equal or
+   * one of the dimensions is 1.
```

Comment:
let's merge these 3 lines into one. Currently it looks artificial.

Reply:
Merged.